### ISSUE TITLE
The Google Colab file doesn't have enough explanation of the figures.

### ISSUE THREAD

**Issue by shammun** [shammun/clim_680_hw#2]
I plotted different graphs but didn't include interpretation of these. There was no clear analysis but rather just graphical representation. My plan is to do this in future.

**Comment by luisortizuriarte** [comment]
Right! You might want to start thinking about a specific question to try to answer with your data.
